tests/ui/test_ui_tools.py

```diff
@@ -1937,17 +1937,17 @@ def test_keypress_EDIT_MESSAGE(self, mocker, message_fixture, widget_size,
                     <p>C</p>
                 </blockquote>
                 <p>D</p>""",
-             ("{} {} {} A\n\n"
-              "{} {} B\n\n"
-              "{} C\n\n"
```

Comment:
Could you explain why the newlines are removed?

Reply:
> blank lines after multi-quotes, eg. no response to a quote and quoted again -> extra blank line per quote level

This is for addressing the first point of your review above.

Are those extra newlines important?